.eslintrc

```diff
@@ -9,6 +9,8 @@
   "extends": "eslint:recommended",
   "rules": {
       "no-unused-vars": ["error", { "vars": "all", "args": "none", "ignoreRestSiblings": true }],
-      "no-constant-condition": ["error", { "checkLoops": false }]
+      "no-constant-condition": ["error", { "checkLoops": false }],
+      "no-var": "error",
+      "prefer-const": "error"
```

Comment:
As I mentioned earlier, I do not agree with this rule. IMO `const` should mark that things are conceptually constant, not just happened to never be reassigned.

Reply:
That's impossible to enforce.
Should it be just everything let then?

Reply:
Again, `no-var` is fine and good but `prefer-const` should not be enabled.